api/src/handlers.rs

```diff
@@ -513,6 +514,7 @@ impl Handler for ChainCompactHandler {
 /// Gets block headers given either a hash or height.
 /// GET /v1/headers/<hash>
 /// GET /v1/headers/<height>
+/// GET /v1/headers/byheight/start_height=200&max=100
```

Comment:
More idiomatic rest api is to use `GET /v1/headers` (without a hash or a height) for listing all headers and control `start_height` and `max` via url parameters.

Reply:
makes sense.
```
/v1/headers/<hash>
/v1/headers/<height>
/v1/headers?start_height=200&max=100
```
I will resubmit the pr.